internal/storage/local/local.go

```diff
@@ -0,0 +1,130 @@
+package local
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	strg "github.com/offen/docker-volume-backup/internal/storage"
+	t "github.com/offen/docker-volume-backup/internal/types"
+	u "github.com/offen/docker-volume-backup/internal/utilities"
+	"github.com/sirupsen/logrus"
+)
+
+type LocalStorage struct {
+	*strg.StorageBackend
+}
+
+// Specific init procedure for the local storage provider.
+func InitLocal(c *t.Config, l *logrus.Logger, s *t.Stats) *strg.StorageBackend {
+	a := &strg.StorageBackend{
+		Storage: &LocalStorage{},
+		Name:    "Local",
+		Logger:  l,
+		Config:  c,
```

Comment:
I would prefer if we wouldn't pass down the entire config object, but only the few values consumed by the specific storage implementation. It doesn't really need to know about all of the other details.

Other values like retention that apply to all backends could also be arguments for the functions.